### ISSUE TITLE
"Zend_Mail_Protocol_Exception: Could not open socket" Error after update to Pimcore 4 ($mail->send()

### ISSUE THREAD

**Issue by dasraab** [pimcore/pimcore#598]
# Bug Report
- since update to Pimcore 4 i´m getting a "could not open socket" error while trying to send mail

$mail = new Mail;
$mail->setIgnoreDebugMode(true);
$mail->setDocument(34);
$mail->setTo("xxx@gmail.com");
$mail->setParams(array());
$mail->send();

this seems to work with older pimcore versions in the same environment. do you have any ideas?

**Comment by brusch** [comment]
I've tested the snippet from above (with my gmail address) and everything worked just fine, so I don't think this is a problem in general. 

Are you using a custom email (SMTP) backend (system settings)? 
If so: are you able to connect to the server using telnet (from the same machine)? 

The error is thrown here: 
https://github.com/zendframework/zf1/blob/210190dab599e2897220648c9040bce9ff76f21f/library/Zend/Mail/Protocol/Abstract.php#L271

So basically PHP isn't able to open a socket to the specified backend server.
I don't think this is related to pimcore. To debug it, put a breakpoint to the line in question, in the `$remote` variable should be the most interesting part. 

If we can do anything further -> let me know.
